package.json

```diff
@@ -61,7 +61,7 @@
     "babel-polyfill": "^6.26.0",
     "brace": "^0.10.0",
     "datatables.net": "~1.10.12",
-    "datatables.net-bs": "~1.10.12",
+    "datatables.net-bs": "^2.1.1",
```

Comment:
iirc, "datatables.net-bs" is the bootstrap styling for datatables. Are you sure bumping this version without changing the matching "datatables.net" version works?

Reply:
I did confirm it works (nothing seems to fail and the page seemed to render correctly).
I do hope we will migrate away from datatables sooner than later anyway..

Reply:
This specific version is marked as deprecated on NPM. Are you sure this is the correct version? https://www.npmjs.com/package/datatables.net-bs lists 1.10.16 as the current version.